tests/test_lif.py

```diff
@@ -291,6 +291,42 @@ def test_lif_firing_rate():
     assert layer.firing_rate > 0
     assert layer.firing_rate == spikes.sum() / (batch_size * time_steps * n_neurons)
 
+@pytest.mark.parametrize("tau_syn", [20.0, None])
+def test_lif_norm_input_with_synapse(tau_syn):
+    batch_size, time_steps, n_neurons = 5, 10, 5
+    # Different tau_mem between 10 and 100
+    tau_mem = (torch.rand(n_neurons) * 90) + 10
+    input_current = torch.rand((batch_size, time_steps, n_neurons))
+    # Very high spike threshold
```

Comment:
Generally, comments are the most helpful when they explain WHY something is done. Someone reading this code can see that a spike threshold of 1e6 is very high. But they don't know why you chose it

Reply:
Okay, good point. I will change that.